### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by lorc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+
+- `flash_writer` - mandatory - defines default Flash Writer filename.
+
+- `baud` - optional - defines baud rate used to communicate with the
+  board. Default value is 115200.
+
+- `sup_baud` - optional - defines "speed-up" baud rate. `rcar_flash`
+  will try to use `sup` command to increase communication speed with
+  the board. This might increase flashing speed considerably. There is
+  no default value, is different boards allow different speeds.
```
Comment (rshym) [inline README.md:324]: Seems like `value, AS different`, but I'm not sure.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+- `cpld_profile` - optional - name of CPLD profile (check the previous
+  section) which should be used to communicate with the board.
+
+- `ipls` - mandatory - list of bootloaders that can be flashed to this
+  board. Each entry should have the following options:
+    - `file` - default file name for said bootloader
+	- `flash_addr` - address in flash memory where to write this bootloader
+	- `flash_target` - which "flash_target" to use while writing this
+      bootloader. Flash targets are describes in the one of the
```
Comment (rshym) [inline README.md:334]: `are describeD`?

**Review by rshym — APPROVED** [review]
Looks really cool!
I just found a few minor misprints.

Reviewed-by: Ruslan Shymkevych <<ruslan_shymkevych@epam.com>>

**Comment by lorc** [comment]
Thank you for the review. A fixed your comments and added a couple of paragraphs. May I ask you to take another look before I hit that "Merge" button?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+# ./rcar_flash.py flash -b <board-name> -s /dev/ttyUSB0 -p <path-to-loader-files> all
+```
+
+Replace `/dev/ttyUSB0` with the correct USB serial port name. In this
+mode `rcar_flash` will start writing bootloaders right away.
+
+
+#### Flashing the board that is in serial download mode
+
+If for `rcar_flash` does not support CPLD control for your board or
```
Comment (rshym) [inline README.md:97]: Seems `for` is not needed in "If for `rcar_flash`"

File: README.md [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+  <https://github.com/renesas-rcar/flash_writer>. Optional
+  `[flashwriter.mot]` parameter allows you to override default flash
+  writer name. Use this option only if your board is already in serial
+  download mode and awaits for flash-writer to be sent, or if you are
+  using `-c` parameter, in this case `rcar_flash` will put the board
+  into serial download mode for you`.
+
+- `-p/--path PATH` - path to bootloader files. By default `rcar_flash`
+  tries to find bootloader files in the current working directory. You
+  can provide another path (for example, `deploy` dir of you Yocto
```
Comment (rshym) [inline README.md:211]: `R` is missed in `dir of youR Yocto`.

**Pull request merged**